python/lsst/sdm/schemas/apdb.yaml

```diff
@@ -1967,3 +1227,1110 @@ tables:
+    description: Cartesian heliocentric Y coordinate at light-emission time (ICRS).
+    datatype: float
+    ivoa:unit: AU
+    ivoa:ucd: pos.cartesian.y;pos.heliocentric
+  - name: helio_z
+    description: Cartesian heliocentric Z coordinate at light-emission time (ICRS).
+    datatype: float
+    ivoa:unit: AU
+    ivoa:ucd: pos.cartesian.z;pos.heliocentric
+  - name: helio_vx
+    description: Cartesian heliocentric X velocity at light-emission time (ICRS).
+    datatype: float
+    ivoa:unit: km/s
+    ivoa:ucd: phys.veloc;pos.cartesian.x
+  - name: helio_vy
+    description: Cartesian heliocentric Y velocity at light-emission time (ICRS).
+    datatype: float
+    ivoa:unit: km/s
+    ivoa:ucd: phys.veloc;pos.cartesian.y
+  - name: helio_vz
+    description: Cartesian heliocentric Z velocity at light-emission time (ICRS).
+    datatype: float
+    ivoa:unit: km/s
+    ivoa:ucd: phys.veloc;pos.cartesian.z
+  - name: helio_vtot
+    description: "The magnitude of the heliocentric velocity vector, sqrt(vx*vx + vy*vy + vz*vz)."
+    datatype: float
+    ivoa:unit: km/s
+    ivoa:ucd: phys.veloc;pos.heliocentric
+
+  - name: topo_x
+    description: Cartesian topocentric X coordinate at light-emission time (ICRS).
+    datatype: float
+    ivoa:unit: AU
+    ivoa:ucd: pos.cartesian.x
+  - name: topo_y
+    description: Cartesian topocentric Y coordinate at light-emission time (ICRS).
+    datatype: float
+    ivoa:unit: AU
+    ivoa:ucd: pos.cartesian.y
+  - name: topo_z
+    description: Cartesian topocentric Z coordinate at light-emission time (ICRS).
+    datatype: float
+    ivoa:unit: AU
+    ivoa:ucd: pos.cartesian.z
+  - name: topo_vx
+    description: Cartesian topocentric X velocity at light-emission time (ICRS).
+    datatype: float
+    ivoa:unit: km/s
+    ivoa:ucd: phys.veloc;pos.cartesian.x
+  - name: topo_vy
+    description: Cartesian topocentric Y velocity at light-emission time (ICRS).
+    datatype: float
+    ivoa:unit: km/s
+    ivoa:ucd: phys.veloc;pos.cartesian.y
+  - name: topo_vz
+    description: Cartesian topocentric Z velocity at light-emission time (ICRS).
+    datatype: float
+    ivoa:unit: km/s
+    ivoa:ucd: phys.veloc;pos.cartesian.z
+  - name: topo_vtot
+    description: "The magnitude of the topocentric velocity vector, sqrt(vx*vx + vy*vy + vz*vz)."
+    datatype: float
+    ivoa:unit: km/s
+    ivoa:ucd: phys.veloc
+
+  - name: diaDistanceRank
+    description: >-
+      The rank of the diaSourceId-identified source in terms of its
+      closeness to the predicted SSO position.  If diaSourceId is the
+      nearest DiaSource to this SSO prediction, diaSourceDistanceRank=1
+      would be set. If it is the second nearest, it would be 2, etc.
+    datatype: int
+
+
+  constraints:
+  - name: fk_SSSource_DiaSource
+    description: Link an SSSource to its associated DiaSource
+    '@type': ForeignKey
+    columns:
+    - '#SSSource.diaSourceId'
+    referencedColumns:
+    - '#DiaSource.diaSourceId'
```

Comment:
This sort of implies that SSSource and DiaSource tables are in the same database? I know this is only a logical model, but some tools may assume that.

Reply:
Will `SSSource` exist in PPDB only?